zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -72,14 +72,15 @@ public ZkClientUriDomainMappingHelper(ZooKeeperServer zks) {
 
     this.rootPath =
         X509AuthenticationConfig.getInstance().getZnodeGroupAclClientUriDomainMappingRootPath();
+    LOG.debug("ZkClientUriDomainMappingHelper::ClientUriDomainMapping Client URI domain mapping root path: {}", this.rootPath);
```

Comment:
Should we make these all `INFO` level logs? Would this be useful in production?

Reply:
Sure i feel it's safe to assume that there would in order of 10s applications per domain name and it would be onetime parsing hence should not bloat logs.